server/audit_math/macro.py

```diff
@@ -252,7 +264,10 @@ def compute_risk(
 
         taint = e_p / u_p
 
-        p *= (1 - 1 / U) / (1 - taint)
+        if taint == 1:
+            p *= 1
```

Comment:
ok maybe im missing something but what's the point of multiplying by 1?

Reply:
if the taint is one, then the p-value is undefined since the KM denominator goes to zero. I suppose the correct thing to do would be to use Decimal's `inf` value, because the p-value goes to infinity in this case. Essentially we're trying to say that if we ever observe a batch that obtains its maximum possible error, then something has truly gone horribly wrong and we need to do a full hand recount.

Reply:
Multiplying p by 1 here is approximately equivalent to the p-value blowing up, but not exactly. I can change it appropriately.

Reply:
cool, that all makes sense to me except why multiplying by 1 blows up the p-value. in math, doesn't multiplying by 1 usually not change the number?

Reply:
I fixed this by setting p to `Decimal('inf')`